app/models/course.rb

```diff
@@ -283,6 +283,10 @@ def scholarship_amount
     dfe_subjects&.first&.scholarship_amount
   end
 
+  def to_s
+    "#{name} (#{course_code})"
```

Comment:
Just wondering provider code as well since that's the unique combo?

Reply:
I'm just following the pattern used in Publish:

![image](https://user-images.githubusercontent.com/23801/59803494-851ccf00-92e3-11e9-8ed1-59bc5ba35010.png)

Also it'll just look redundant and noisy in mcb:

![image](https://user-images.githubusercontent.com/23801/59803534-a7165180-92e3-11e9-908e-9ac6384cd116.png)

So I'd prefer to leave as is.